valheim-server

```diff
@@ -70,10 +70,12 @@ ensure_permissions_after_start() {
 wait_for_server_listening() {
     while :; do
         if server_is_listening; then
+            post_server_listening_hook
             update_server_status running
             debug "Server is now listening on UDP port $SERVER_PORT"
             break
         else
+            pre_server_listening_hook
```

Comment:
This should be outside the while loop. Otherwise we're calling the hook every five seconds.